energyplus_iddidf/__main__.py

```diff
@@ -0,0 +1,4 @@
+from energyplus_iddidf.cli import main_cli
+
+
+main_cli()
```

Comment:
Provides an entry point using `-m`, which is how some folks expect to use the installed library.  As an example, this can be used like:

```
$ ./bin/python -m energyplus_iddidf --summarize_idd_object PlantLoop /path/to/Energy+.idd 
{
  "message": "Everything looks OK",
  "content": {
    "searched_object_name": "PLANTLOOP",
    "field": [
      "A1 : Name",
      "A2 : Fluid Type",
      "A3 : User Defined Fluid Type",
      "A4 : Plant Equipment Operation Scheme Name",
      "A5 : Loop Temperature Setpoint Node Name",
      "N1 : Maximum Loop Temperature",
      "N2 : Minimum Loop Temperature",
      "N3 : Maximum Loop Flow Rate",
      "N4 : Minimum Loop Flow Rate",
      "N5 : Plant Loop Volume",
      "A6 : Plant Side Inlet Node Name",
      "A7 : Plant Side Outlet Node Name",
      "A8 : Plant Side Branch List Name",
      "A9 : Plant Side Connector List Name",
      "A10 : Demand Side Inlet Node Name",
      "A11 : Demand Side Outlet Node Name",
      "A12 : Demand Side Branch List Name",
      "A13 : Demand Side Connector List Name",
      "A14 : Load Distribution Scheme",
      "A15 : Availability Manager List Name",
      "A16 : Plant Loop Demand Calculation Scheme",
      "A17 : Common Pipe Simulation",
      "A18 : Pressure Simulation Type",
      "N6 : Loop Circulation Time"
    ]
  }
}
```

Pretty cool!